ml-agents/mlagents/trainers/tests/test_bcmodule.py

```diff
@@ -0,0 +1,89 @@
+import unittest.mock as mock
+import pytest
+import mlagents.trainers.tests.mock_brain as mb
+
+import numpy as np
+import tensorflow as tf
+import yaml
+import os
+
+from mlagents.trainers.ppo.models import PPOModel
+from mlagents.trainers.ppo.trainer import discount_rewards
+from mlagents.trainers.ppo.policy import PPOPolicy
+from mlagents.envs import UnityEnvironment
+
+
+@pytest.fixture
+def dummy_config():
+    return yaml.load(
```

Comment:
nit: I switched the tests to use `yaml.safe_load` in https://github.com/Unity-Technologies/ml-agents/pull/2124 instead to clean up some warnings